<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1660</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Play Tab</string>
       </attribute>
       <widget class="QPushButton" name="BrowsepushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>93</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="SettingsgroupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>420</y>
          <width>571</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Settings </string>
        </property>
        <widget class="QLabel" name="Volumelabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
        <widget class="QPushButton" name="PlayPausepushButton">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>20</y>
           <width>93</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Play/Pause</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ShowSpectrogrampushButton">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>20</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Show Spectrogram</string>
         </property>
        </widget>
        <widget class="QSlider" name="VolumeverticalSlider">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>22</width>
           <height>111</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Instrument1verticalSlider">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>22</width>
           <height>111</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Instrument2verticalSlider">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>50</y>
           <width>22</width>
           <height>111</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="Instrument3verticalSlider">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>22</width>
           <height>111</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="Instrument1label">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Instrument 1</string>
         </property>
        </widget>
        <widget class="QLabel" name="Instrument2label">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Instrument 2</string>
         </property>
        </widget>
        <widget class="QLabel" name="Instrument3label">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>20</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Instrument 3</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>711</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Composing Tab</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1660</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
